.github/spellcheck/.wordlist.txt

```diff
@@ -301,17 +329,20 @@ ValidatingWebhookConfiguration
 VolumeAttachment
 WDS
 WDSes
+WDSs
```

Comment:
This is for one item in olddocs/, right? Since those are not maintained, maybe we could skip them in the spell-checking?